### PR TITLE
feat :  add blackman_window to PyTorch frontend

### LINKED ISSUE #24983: blackman_window

### PR DESCRIPTION
**Pull request by Muhammad-Hamza12**
Close #24983

### CODE REVIEW

File: ivy/functional/frontends/torch/spectral_ops.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import ivy
-from ivy.functional.frontends.torch.func_wrapper import to_ivy_arrays_and_back
```
Comment (paulaehab) [inline ivy/functional/frontends/torch/spectral_ops.py:2]: Could you please return this line

**Comment by paulaehab** [comment]
Hey @Muhammad-Hamza12  Nice work thanks for contributing I noticed that the jax and paddle backend tests fail because of "cos_vml_cpu" not implemented for 'Half' try to make unsupported dtypes for each of them the half 
Also, I want you to uncommit all changes in any other file except the function file and test files 
after you cone all of these check the tests are passing or not

File: ivy/functional/frontends/torch/spectral_ops.py (hunk 1 above)
Comment (Muhammad-Hamza12) [reply ivy/functional/frontends/torch/spectral_ops.py:2]: yes i have returned this line.

**Comment by Muhammad-Hamza12** [comment]
Hi @paulaehab ,

I have made the suggested changes. Following are the keys issues addressed,

1.  All of the tests are passing.
2.  Unnecessary files removed.
3.  I have added decorator to support float types other than half.
4.  contributing.md  file is showing changes due to 'lint code' commit.
5.  i have made changes to the file 'ivy/functional/backends/jax/experimental/creation.py' and edited blackman_Window function implementation to typecast the returned array into dtype passed as argument to the function.
6.  i have made changes to the file 'ivy/functional/backends/tensorflow/experimental/creation.py' and edited blackman_Window function implementation to typecast the returned tf tensor/variable  into dtype passed as argument to the function.


Kindly review my PR again,


Looking forward to hearing from you.

**Pull request merged**